libs/pageserver_api/src/shard.rs

```diff
@@ -411,6 +411,12 @@ impl ShardIdentity {
             String::new()
         }
     }
+
+    /// Convenience for checking if this identity is the 0th shard in a tenant,
+    /// for special cases on shard 0 such as ingesting relation sizes.
+    pub fn is_zero(&self) -> bool {
```

Comment:
I prefer to have more meaningful name for this method which explains special meaning of shard 0.
In this case we should not explain this role in any place where this method is used

Reply:
Yeah, there are a few helpers here where the meaning is a bit obscure.  I've created https://github.com/neondatabase/neon/issues/6072 to make a pass over shard.rs and try and come up with the most maintainable names we can